sunpy/coordinates/transformations.py

```diff
@@ -176,6 +176,57 @@ def hgs_to_hcc(heliogcoord, heliocframe):
     return heliocframe.realize_frame(representation)
 
 
+@frame_transform_graph.transform(FunctionTransform, HelioProjective,
+                                 HelioProjectiveRadial)
+def hpc_to_hpr(hpcframe, hprframe):
+    """
+    Transform from the hpcframe to a hprframe
+    """
+    # Put it in rads for np
+    lon = np.deg2rad(hpcframe.Tx)
+    lat = np.deg2rad(hpcframe.Ty)
+    # Elongation calc:
+    # Get numerator and denomenator for atan2 calculation
+    top = np.cos(lat)*np.cos(lon)
+    btm = np.sqrt((np.cos(lat)**2)*(np.sin(lon)**2) + (np.sin(lat)**2))
+    el = np.arctan2(top,btm)
```

Comment:
Looking at Eq19 from Thompson, and his notation that arg(x,y) = arctan(y/x), I think the top and bottom need swapping round in this call to arctan2. Same applies to the pa calculation on Line 196. However, when I tried this it returned ValueError: Latitude angle(s) must be within -90 deg <= angle <= 90 deg